packages/components/src/components/multi-select/multi-select/multi-select.tsx

```diff
@@ -181,7 +181,7 @@ export class MultiSelect {
     document.addEventListener('mousedown', this.onClickOutside, true);
     this.form = getClosestHTMLElement(this.host, 'form');
     this.isWithinForm = !!this.form;
-    this.isNativePopoverCase = detectNativePopoverCase(this.host, false);
+    this.isNativePopoverCase = !!detectNativePopoverCase(this.host, false);
```

Comment:
boolean casting shouln't be necessary as `detectNativePopoverCase` returns a boolean